docs/product/issues/issue-details/index.mdx

```diff
@@ -102,6 +102,28 @@ If you’ve enabled [Session Replay](/product/explore/session-replay/), you’ll
 
 ![Session Replay](./img/issue-replay.png)
 
+## Feature Flags
+If you’ve enabled a feature flag integration in your SDK you’ll have access to the feature flag table. If any flags were captured after the Sentry SDK's initialization and before the error event occurred, the table will contain rows of flag, evaluation-result pairs.
```

Comment:
There are two types of integrations you can configure to integrate feature flag data into your error events.

The first is to configure runtime feature flag evaluation tracking in the sentry SDK. Doing this will give you access to the feature flag table which shows any flags that were evaluated after the Sentry SDK's initialization and before the error event occurred. You will see rows of flag, evaluation-result pairs.

The second is to configure feature flag change tracking. This will enable a timeline of feature flag changes inside the releases graph. These annotations can help you diagnose when a feature flag change is related to a new issue.

When both of these integration are enabled you will also see suspect feature flags highlighted inside the feature flag table. Feature flags are suspect when their modification occurs suspiciously close to the initial detection of a new issue. Suspect feature flags are displayed are given a yellow highlight within the feature flag table.